def foo():
    return 10 + 'ali'
try:
    foo()
except:
    print('An exception occured...')
print('Continues...')
